src/main-menu.ts

```diff
@@ -1,24 +1,98 @@
-import { LitElement, PropertyValueMap, css, html } from "lit";
-import { customElement, state } from "lit/decorators.js";
+import { LitElement, css, html } from "lit";
+
+import { customElement, property, state } from "lit/decorators.js";
 import { game } from "./game";
-import { AudioController } from "./audio/audio_controller";
 
 @customElement("main-menu")
 export class MainMenu extends LitElement {
   static styles = css`
     :host {
-      width: 100vw;
-      height: 100vh;
+      width: 100%;
+      aspect-ratio: 16 / 9;
+      position: relative;
       display: flex;
       align-items: center;
       justify-content: center;
       flex-direction: column;
+      background: rgb(240, 200, 98);
+    }
+    h1 {
+      font-family: "Poppins", sans-serif;
+      font-size: 12vw;
+      position: absolute;
```

Comment:
Lol feels weird to dictate the position of all H1's but ok

Reply:
Updated to use a class